tests/std/tests/P0088R3_variant_msvc/test.cpp

```diff
@@ -909,6 +910,33 @@ namespace msvc {
             }
         }
     } // namespace assign_cv
+
+    namespace gh4959 {
+        // Test GH-4959 "P0608R3 breaks flang build with Clang"
+        // Constraints on variant's converting constructor and assignment operator templates reject arguments of the
+        // variant's type, but did not short-circuit to avoid evaluating the constructibility constraint. For this
+        // program, the constructibility constraint is ill-formed outside the immediate context when determing if
+        // variant<optional<GenericSpec>> can be initialized from an rvalue of the same type.
+
+        template <typename... RVREF>

```

Comment:
Nitpick: This is a macro-like identifier which we usually avoid in test code (`A` is okay but this is very shouty).